subsys/usb/common/include/usb_uvc.h

```diff
@@ -24,12 +24,18 @@
 /* Video Class-Specific Request Codes */
 #define UVC_SET_CUR					0x01
 #define UVC_GET_CUR					0x81
+#define UVC_SET_CUR_ALL				0x11
 #define UVC_GET_MIN					0x82
 #define UVC_GET_MAX					0x83
 #define UVC_GET_RES					0x84
 #define UVC_GET_LEN					0x85
-#define UVC_GET_INFO					0x86
+#define UVC_GET_INFO				0x86
```

Comment:
Just small detail:
Is indentation change needed? If not best to keep the same indentation everywhere.
You might need to configure the  text editor to use "tabs 8 character wide".